src/content/app-platform/namespace-configuration/index.md

```diff
@@ -0,0 +1,78 @@
+---
+linkTitle: App target namespace metadata
+title: How to annotate an app's target namespace via its App CR
+description: Configuring metadata for the target namespace of an app via its app CR so it can be used by other tools such as service meshes.
+weight: 20
+menu:
+  main:
+    parent: app-platform
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+aliases:
+  - /reference/app-configuration/
```

Comment:
```suggestion
```

Reply:
🙈 Sorry, it was left over from copy & paste.